wannierberri/__parallel.py

```diff
@@ -20,26 +20,15 @@ class Parallel():
 
     def __init__(
             self,
-            method=None,
             num_cpus=0,
             npar_k=0,
             ray_init={},  # add extra parameters for ray.init()
             cluster=False,  # add parameters for ray.init() for the slurm cluster
             progress_step_percent=1):
 
-        if method is None:
-            if num_cpus <= 0:
-                method = "serial"
-            else:
-                method = "ray"
-
-        self.method = method
+        self.method = "ray" if num_cpus>0 else "serial"
```

Comment:
This means ray requires manual num cpus input. Maybe allow num cpus = “auto”? Also num cpus 0 for serial seems a bit unintuitive, I think None is better.

Reply:
To be more explicit, now that's two separate classes : `Parallel()` and its child `Serial()`